build.gradle

```diff
@@ -1,4 +1,4 @@
-plugins {
+    plugins {
```

Comment:
Should the indentation here be no space instead?

Reply:
Fixed. Thanks for pointing out!